### DISCUSSION TITLE
sign in Error in vercel

### DISCUSSION THREAD

**Discussion by Sakibalhasan164** [vercel/vercel#5774] [Help]
I a trying to Sign in to vercel ..but I a getting a error every time .Hare's the error looks like:
"This user account is blocked. Contact support@vercel.com for more information."

I am using a VPN to acess Vercel because i am unable to acess this site with one..Is this the reason for the Error?

**Comment by magikadev** [comment]
Same error here.

But I didn't use VPN.

**Reply by paulogdm** [reply]
Please reach out to our support channel so we can give you more information about the block.

**Reply by magikadev** [reply]
Can you please send me the link to support channel?

**Reply by magikadev** [reply]
I have never used vercel before. I can't understand why my account is blocked...

**Reply by paulogdm** [reply]
You can use `support@vercel.com`
